lib/gateways/snapshot-gateway.js

```diff
@@ -72,6 +72,30 @@ class SnapshotGateway {
     );
     return { snapshots };
   }
+
+  async save({ snapshot }) {
+    if (!snapshot) throw new ArgumentError('snapshot cannot be null.');
+
+    const request = {
+      TableName: this.tableName,
+      Key: { id: snapshot.id },
+      ConditionExpression: 'attribute_exists(id)',
+      UpdateExpression: [
+        'set assets = :a',
+        'notes = :n',
+        'vulnerabilities = :v'
+      ].join(', '),
```

Comment:
Is this just to make it look nicer in the code?

Reply:
pretty much